tests/vtr/lutff-pair/README.rst

```diff
@@ -0,0 +1,46 @@
+LUT with FlipFlop Example
++++++++++++++++++++++++++
+
+An example of the classical LUT with FlipFlop pair shown in |fig31|_ demonstrating the `<pack_pattern>` tag.
+
```

Comment:
Why did the line;
> This is shown in `Figure 28` of the [Architecture Reference section of the Verilog to Routing Documentation](https://docs.verilogtorouting.org/en/latest/arch/reference) and reproduced below;

disappear?

Reply:
Looks like that README was already modified in https://github.com/SymbiFlow/python-symbiflow-v2x/pull/71 where I started off.

Reply:
I've added the link back.